guide/how-it-works/node.md

```diff
@@ -85,11 +99,13 @@ A local node runs a node on your local device using software such as [Bitcoin Co
    layout = "float-left"
 %}
 
-A hosted node is when a user runs their own node on a trusted third-party server. This puts full trust in the third-party that they will not tamper with your block data. This also exposes your transaction history to them, which is a potential privacy risk.
+Hosted nodes are run on third-party servers and are common with merchants using [BTCPay Server](https://btcpayserver.org/). While you have some degree of control over a hosted node, using a hosted node can expose your transaction history to the hosting provider and involves trusting them not to tamper with your transaction data.
+
+[Greenlight](https://blockstream.com/lightning/greenlight/) and [Validating Lightning Signer (VLS)](https://gitlab.com/lightning-signer) are experimental ways to run a hosted lightning node whilst users keep custody of their private keys.
```

Comment:
I'm unsure if voltage is custodial, they claim otherwise but not sure how it works. I will err on the side of custodial for now. They aren't open source either, tried to focus on examples that where.

Added saome details about the pros of hosting nodes